Library/Homebrew/cmd/cleanup.rb

```diff
@@ -64,12 +64,13 @@ def cleanup_formula f
   end
 
   def cleanup_keg keg
+    size = `/usr/bin/du -sh #{keg} | /usr/bin/awk -F'[^0-9]*' '{print $2}'`.strip
```

Comment:
Agreed, massively. I just wasn't sure how to grab folder sizes from Ruby given folders are basically, well, nothingness in themselves, it's the contents that contain the size, and I wasn't overly familiar with how Ruby handles sizing the entire contents of folders. I'm not even sure what I just said made much sense, hopefully it did. I'll look at the ` info.rb ` logic :smile_cat: